compose/ui/ui-test/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -443,4 +455,11 @@ actual sealed interface ComposeUiTest : SemanticsNodeInteractionsProvider {
     actual fun registerIdlingResource(idlingResource: IdlingResource)
     actual fun unregisterIdlingResource(idlingResource: IdlingResource)
     actual fun setContent(composable: @Composable () -> Unit)
+    val lifecycleRegistry: LifecycleRegistry
```

Comment:
I'd avoid exposing it to public. Maybe expose `sendLifecycleEvent` like we have for input instead?

Reply:
Yeah, change, but I'd leave it here for now as this class is marked with `@ExperimentalTestApi`?

Reply:
I vote for `sendLifecycleEvent`